src/app/Plans/components/Wizard/WizardComponent.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import WizardStepContainer from './WizardStepContainer';
+import { Wizard } from '@patternfly/react-core';
+import Review from './Review';
+import GeneralForm from './GeneralForm';
+import { Provider } from '@app/Providers/types';
+
+interface IWizardComponent {
+  isOpen: boolean;
+  onHandleWizardClose: () => void;
+  sourceProviders: Provider[];
+  targetProviders: Provider[];
+}
+
+const WizardComponent: React.FunctionComponent<IWizardComponent> = ({
+  isOpen,
+  onHandleWizardClose,
+  sourceProviders,
+  targetProviders,
+}: IWizardComponent) => {
+  enum stepId {
+    General = 1,
+    FilterVMs,
+    SelectVMs,
+    StorageMapping,
+    NetworkMapping,
+    Hooks,
+    Review,
+  }
+
+  const steps = [
+    {
+      id: stepId.General,
+      name: 'General',
+      component: (
+        <WizardStepContainer title="General Settings">
+          <GeneralForm sourceProviders={sourceProviders} targetProviders={targetProviders} />
+        </WizardStepContainer>
+      ),
+      enableNext: true,
+    },
+    {
+      name: 'VM Selection',
+      steps: [
+        {
+          id: stepId.FilterVMs,
+          name: 'Filter VMs',
+          component: (
+            <WizardStepContainer title="Filter VMs">
+              <div>Filter VMs</div>
+            </WizardStepContainer>
+          ),
+          enableNext: true,
+        },
+        {
+          id: stepId.SelectVMs,
+          name: 'Select VMs',
+          component: (
+            <WizardStepContainer title="Select VMs">
+              <div>Select VMs</div>
+            </WizardStepContainer>
+          ),
+          enableNext: true,
+        },
+      ],
+    },
+    {
+      id: stepId.StorageMapping,
+      name: 'Storage Mapping',
+      component: (
+        <WizardStepContainer title="Storage Mapping">
+          <div>Storage mapping</div>
+        </WizardStepContainer>
+      ),
+      enableNext: true,
+    },
+    {
+      id: stepId.NetworkMapping,
+      name: 'Network Mapping',
+      component: (
+        <WizardStepContainer title="Network Mapping">
+          <div>Network mapping</div>
+        </WizardStepContainer>
+      ),
+      enableNext: true,
+    },
+    {
+      id: stepId.Hooks,
+      name: 'Hooks',
+      component: (
+        <WizardStepContainer title="Hooks">
+          <div>Hooks</div>
+        </WizardStepContainer>
+      ),
+      enableNext: true,
+    },
+    {
+      id: stepId.Review,
+      name: 'Review',
+      component: <Review />,
+      nextButtonText: 'Finish',
+    },
+  ];
+
+  const handleClose = () => {
+    onHandleWizardClose();
```

Comment:
The `onHandleWizardClose` naming is a bit redundant, I think we can just call the prop `onClose` and pass it directly into `<Wizard>` instead of having the `handleClose` function in the middle.